cromwell_tools/tests/test_cromwell_tools.py

```diff
@@ -234,6 +236,56 @@ def test_validate_cromwell_label_on_valid_labels_str_object(self):
     def test_validate_cromwell_label_on_valid_labels_bytes_object(self):
         self.assertIsNone(cromwell_tools.validate_cromwell_label(json.dumps(self.valid_labels).encode('utf-8')))
 
+    def test_localize_file_https(self):
+        temporary_directory = tempfile.mkdtemp()
+        # grab this file from the master branch of the cromwell-tools repository
+        target = ('https://raw.githubusercontent.com/broadinstitute/cromwell-tools/master/'
```

Comment:
why master branch instead of a tagged version? I'm not sure if master in the future will still have this file as we expect here.

Reply:
Fair enough. I figured it was safe for the file to retrieve itself. Changed.